hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieClusteringConfig.java

```diff
@@ -61,6 +61,8 @@ public class HoodieClusteringConfig extends HoodieConfig {
       "org.apache.hudi.client.clustering.run.strategy.SparkSortAndSizeExecutionStrategy";
   public static final String SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY =
       "org.apache.hudi.client.clustering.run.strategy.SparkConsistentBucketClusteringExecutionStrategy";
+  public static final String SINGLE_SPARK_CONSISTENT_BUCKET_EXECUTION_STRATEGY =
+      "org.apache.hudi.client.clustering.run.strategy.SingleSparkConsistentBucketClusteringExecutionStrategy";
```

Comment:
Rename to `SingleSparkJobConsistentHashingExecutionStrategy`

Reply:
> Rename to `SingleSparkJobConsistentHashingExecutionStrategy`

done~